lib/config.go

```diff
@@ -65,3 +65,27 @@ func sourceProfile(p string, from profiles) string {
 	}
 	return p
 }
+
+func GetValueFromProfile(profile string, from profiles, config_key string) (string, string, error) {
```

Comment:
How do you feel about making this a method on the `profiles` type?

Reply:
@nickatsegment the one complication is that `profiles` is not an exported type, which I think makes https://github.com/segmentio/aws-okta/pull/111/files#diff-aed93a53379775027b6616ae133e3815R57 not work?  I think it is cleaner to put it on `profiles`, but should I export the `profiles` type?